src/components/contactForm/ContactForm.js

```diff
@@ -12,21 +10,23 @@ export const ContactForm = ({
 }) => {
   return (
     <form onSubmit={handleSubmit}>
-      {/* name input */}
-    <input type="text" placeholder="name" value={name}
-        onChange={({target}) => {setName(target.value)}}
-        required />
-      {/* phone numer input */}
-      <input type="number" placeholder='phone number'
-       pattern="[1-9][0-9]{2}-[1-9][0-9]{2}-[0-9]{4}" value={phone} onChange={({ target }) => { setPhone(target.value) }} required />
-      
-      {/* ^((\+\d{1,3}(-| )?\(?\d\)?(-| )?\d{1,3})|(\(?\d{2,3}\)?))(-| )?(\d{3,4})(-| )?(\d{4})(( x| ext)\d{1,5}){0,1}$ */}
-      {/* email input  */}
-
-      <input type="text" placeholder='email' 
-        value={email} onChange={({ target }) => { setEmail(target.value) }} required />
-      
-       <button>Submit</button>
-    </form>
+        <label>
+          Contact Name:
+          <input type="text" name="cname" value={cname} onChange={(e)=> setName(e.target.value)}required palceholder="Contact Name"/>

```

Comment:
Please compare your code and only submit code changes that are an improvement. It is good that you are cleaning up code and removing unnecessary comments or incorrect formatting.

However:
- There is a typo in placeholder
- change of name to "cname" affects code throughout the app
- have removed the regex pattern that ensures a telephone number is entered.
- is there a reason why you prefer to export the component as a default export at the bottom of the file, rather than as a named export at the top?